static/css/mozsearch.css

```diff
@@ -1204,11 +1206,7 @@ input::placeholder {
   margin-bottom: 1em;
 }
 
-#svg-preview-checkerboard {
-  margin-inline-start: 0;
-}
-
-#svg-preview-checkerboard:checked + label + a > img {
+#svg-preview-background-checkerboard:checked + label + input + label + input + label + a > img {
```

Comment:
Probably better to use `#svg-preview-background-checkerboard:checked ~ a > img` (and same for the others) as it's simpler and not meaningfully slower tbh. But your call